ui.apps/src/main/content/jcr_root/apps/acs-commons/clientlibs/vendor/angularjs/v1.8.2/all/.content.xml

```diff
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
-    jcr:primaryType="cq:ClientLibraryFolder"
+          jcr:primaryType="cq:ClientLibraryFolder"
           allowProxy="{Boolean}true"
           jsProcessor="[default:none,min:gcc]"
-          categories="[acs-commons.angularjs,acs-commons.angularjs.v1.2.all]"/>
+          categories="[acs-commons.angularjs,acs-commons.angularjs.v1.8.2.all]"/>
```

Comment:
I would make the version specific category only end with 1.8. The micro version is usually not worth pinning. That way micro updates can be done without adjusting the category name.